server/routes/billRoute.js

```diff
@@ -45,18 +45,37 @@ router.post(
 // @desc    Add a dish to the bill
 // @access  Private
 
-/* :id is a placeholder for bill id. This way, we are only able add dishes,
+/* 
+	:id is a placeholder for bill id. This way, we are only able to add dishes
     when we have the bill id.
  */
 router.post(
-	'/dish/:id',
+	'/add_dishes',
```

Comment:
Changed post path because having the bill id visible in the url might be a security problem.
Instead, the bill id could be passed in the request body.